README.md

```diff
@@ -287,15 +287,19 @@ want this to be a k3s server (controlplane):
 variant: flatcar
 version: 1.0.0
 storage:
-  links:
   files:
     # filename needs to be k3s.raw
     - path: /etc/extensions/k3s.raw
       contents:
         source: https://github.com/flatcar/sysext-bakery/releases/download/latest/k3s-v1.29.2+k3s1-x86-64.raw
     - path: /etc/systemd/system/multi-user.target.wants/k3s.service
       target: /usr/local/lib/systemd/system/k3s.service
-      overwrite: true
+      overwrite: false
```

Comment:
@tormath1 I think we should also update the user and group of `/etc/rancher/k3s/k3s.yaml` to avoid calling `kubectl` commands with sudo always. 

I wanted to confirm if you think that, this shouldn't be a part of example as of now?

Reply:
updated and now we are not doing anything with kubeconfig. (user will require to copy/export/link it etc. as you wrote above.)